tests/system/test_query.py

```diff
@@ -528,7 +526,6 @@ def make_entities():
     assert not more
 
 
-@pytest.mark.skip("Requires an index")
```

Comment:
For some reason this skip is still on master. Merge weirdness?

Reply:
Yikes! I'll get that fixed.